client/my-sites/comments/comment/index.jsx

```diff
@@ -141,6 +142,8 @@ export class Comment extends Component {
 				{ isEditMode && (
 					<CommentEdit { ...{ commentId } } toggleEditMode={ this.toggleEditMode } />
 				) }
+
+				{ isPostView && <CommentRepliesList { ...{ siteId, commentParentId: commentId } } /> }
```

Comment:
We should check here to see if we've are already reached nesting depth limit, and if yes move this outside of existing comment card.